.circleci/config.yml

```diff
@@ -390,13 +390,6 @@ jobs:
             FOUNDRY_PROFILE: ci
           working_directory: packages/contracts-bedrock
           no_output_timeout: 15m
-      - run:
```

Comment:
We should bring in this change.

Reply:
Do you mean implement an alternative that actually checks that the deploy configs are working? Very unclear what you mean here

Reply:
that (at least) this part of the diff should be merged. I'm trying to make it clear that I realize that we do want this PR.